events/app.py

```diff
@@ -0,0 +1,113 @@
+import os
+import logging
+import datetime
+import pymongo
+# import pprint
+
+from flask import Flask, request, Response
+
+app = Flask(__name__)
+
+# connect to MongoDB Atlas database
+MONGODB_ATLAS_USERNAME = os.environ.get(
+    "MONGODB_ATLAS_USERNAME")
+MONGODB_ATLAS_PASSWORD = os.environ.get(
+    "MONGODB_ATLAS_PASSWORD")
+MONGODB_ATLAS_CLUSTER_ADDRESS = os.environ.get(
+    "MONGODB_ATLAS_CLUSTER_ADDRESS")
+PYMONGO_URI = "mongodb+srv://{}:{}@{}".format(
+    MONGODB_ATLAS_USERNAME,
+    MONGODB_ATLAS_PASSWORD,
+    MONGODB_ATLAS_CLUSTER_ADDRESS)
+# print("Pymongo URI: {}".format(PYMONGO_URI))
+client = pymongo.MongoClient(PYMONGO_URI)
```

Comment:
This is a different way of building the Mongo client compared with #2. You and Gabe should compare notes, and consider a standard way of handling this. (His has a `if __name__ == '__main__'` block that does the initialization.)

Reply:
done, switched to his method which is needed for testing with mongomock

Reply:
Actually, the previous way I had of connecting to the db in if __name__ == '__main__' will break in Travi-CI, but you aren't seeing that because our test harness isn't set up to test the events directory. I've just gone on my branch and fixed up the database connecting to be all safe for testing, see the latest version at https://github.com/knative-portability/large-events/blob/0a95f7c496bcddbde80124069fd76e7c48ada800/users/app.py

Reply:
I set up the db in a similar way to your current method, and set up testing for the events directory. Things seem to be working at the moment

Reply:
I actually just update things (sorry, I think this is the last one) to be simpler and not use the `global` keyword (which actually shows up as a warning from pylint). If you want I could switch to this later in a separate PR though. https://github.com/mukobi/large-events/blob/users-mongodb-atlas/users/app.py

Reply:
I changed the functions for connecting to the DB to be very similar now.